lib/node_modules/@stdlib/array/complex64/docs/types/index.d.ts

```diff
@@ -195,6 +195,60 @@ type TernaryMapFcn<U> = ( this: U, value: Complex64, index: number, arr: Complex
 */
 type MapFcn<U> = NullaryMapFcn<U> | UnaryMapFcn<U> | BinaryMapFcn<U> | TernaryMapFcn<U>;
 
+/**
+* Reducer function invoked for each element in an array.
+*
+* @returns accumulated result
+*/
+type NullaryReducer<U, V> = ( this: U ) => V;
```

Comment:
```suggestion
type NullaryReducer<U> = () => U;
```

In contrast to other array methods, the `reducer` is not provided a `thisArg`.